src/main/resources/assets/croptopia/lang/de_de.json

```diff
@@ -0,0 +1,282 @@
+{
+  "item.croptopia.artichoke": "Artischocke",
+  "item.croptopia.asparagus": "Spargel",
+  "item.croptopia.bellpepper": "Paprika",
+  "item.croptopia.blackbean": "Schwarze Bohne",
+  "item.croptopia.blackberry": "Brombeere",
+  "item.croptopia.blueberry": "Blaubeere",
+  "item.croptopia.broccoli": "Brokkoli",
+  "item.croptopia.cabbage": "Kohl",
+  "item.croptopia.cantaloupe": "Honigmelone",
+  "item.croptopia.cauliflower": "Blumenkohl",
+  "item.croptopia.celery": "Seleri",
+  "item.croptopia.coffee_beans": "Kaffeebohnen",
+  "item.croptopia.corn": "Mais",
+  "item.croptopia.cranberry": "Cranberry",
+  "item.croptopia.cucumber": "Gurke",
+  "item.croptopia.currant": "Johannisbeere",
+  "item.croptopia.eggplant": "Aubergine",
+  "item.croptopia.elderberry": "Holunderbeere",
+  "item.croptopia.garlic": "Knoblauch",
+  "item.croptopia.grape": "Weintrauben",
+  "item.croptopia.greenbean": "Grüne Bohne",
+  "item.croptopia.greenonion": "Grüne Zwiebel",
+  "item.croptopia.honeydew": "Honigtau",
+  "item.croptopia.hops": "Hopfen",
+  "item.croptopia.kale": "Grünkohl",
+  "item.croptopia.kiwi": "Kiwi",
+  "item.croptopia.leek": "Lauch",
+  "item.croptopia.lettuce": "Salatblatt",
+  "item.croptopia.olive": "Olive",
+  "item.croptopia.onion": "Zwiebel",
+  "item.croptopia.peanut": "Erdnuss",
+  "item.croptopia.pineapple": "Ananas",
+  "item.croptopia.radish": "Radieschen",
+  "item.croptopia.raspberry": "Himbeere",
+  "item.croptopia.rhubarb": "Rhabarber",
+  "item.croptopia.rice": "Reis",
+  "item.croptopia.rutabaga": "Steckrübe",
+  "item.croptopia.saguaro": "Kaktusfrucht",
+  "item.croptopia.spinach": "Spinat",
+  "item.croptopia.squash": "Squash",
```

Comment:
I'd be curious which type of squash this refers to, because technically pumpkins are a squash too.